glue/flagging_script_glue/flagging.py

```diff
@@ -130,12 +124,18 @@ def iso_forest(
     )
     df["sv_anomaly"] = isof.fit_predict(feed)
 
+    # Assign labels for anomalies
     df["sv_anomaly"] = np.select(
         [(df["sv_anomaly"] == -1), (df["sv_anomaly"] == 1)],
         ["Outlier", "Not Outlier"],
         default="Not Outlier",
     )
 
+    # Restore original values for encoded columns
+    for group, le in label_encoders.items():
+        df[group] = le.inverse_transform(df[group])
+
+    # Reset index
```

Comment:
This is a fix that temporarily encodes iso forest groups to numeric if they are not able to be converted to numeric.